api/RTCPeerConnection.json

```diff
@@ -319,7 +319,59 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "addstream_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/addstream_event",
+          "description": "<code>addstream</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "24"
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
Should be 25. I don't have a link, but the general history is that there was no Chrome 24 for Android, and from 25 onward, Chrome Android maintained feature parity with Chrome.

Reply:
@jpmedley - How about webview for this one?

Reply:
If the version number is earlier than 37, webview should be `true` unless you have specific information about the pre-chromium webview. In which case I would appreciate the chance to examine that information. (I should have mentioned this in my comment on the other PR.)